File: C Programming/Competitive Programming/Coding Challenges 6/program5.c
// 5. Write a program which accept number from user and display its table in reverse
// order.
// Input : 2
// Output : 20 18 16 14 12 10 8 6 4 2
// Input : 5
// Output : 50 45 40 35 30 25 20 15 10 5
// Input : -5
// Output : 50 45 40 35 30 25 20 15 10 5

#include<stdio.h>

void TableRev(int iNo)
{
    int i = 10;

    if (iNo<0)
    {
        iNo = -iNo;
    }
    printf ("Output is: ");
    while (i>=1)
    {
        printf(" %d\t",iNo*i);
        i--;
    }
    

}

int main()
{
    int iValue = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    TableRev(iValue);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 11/program5.java
//5.Write a program which accept number from user and return difference between summation of all its factors and non factors.
//Input : 12
//Output : -34 (16 - 50)
//Input : 10
//Output : -29 (8 - 37)

import java.util.Scanner;

class program5
{
    static int FactDiff(int iNo)
    {
        if (iNo < 0)
        {
            iNo = -iNo;
        }

        int i = 1;
        int iTotalSum = 0;
        int iFactSum = 0;

        while (i < iNo)
        {
            iTotalSum = iTotalSum + i;

            if (iNo % i == 0)
            {
                iFactSum = iFactSum + i;
            }

            i++;
        }

        int iNonFactSum = iTotalSum - iFactSum;

        return iFactSum - iNonFactSum;
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        int iRet = 0;

        System.out.print("Enter number ");
        iValue = sobj.nextInt();

        iRet = FactDiff(iValue);

        System.out.print(iRet);

        sobj.close();
    }
}